packages/react/src/forwardRef.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {REACT_USE_REF_TYPE} from 'shared/ReactSymbols';
+
+import warning from 'fbjs/lib/warning';
+
+export default function forwardRef<Props, ElementType: React$ElementType>(
+  renderProp: (props: Props, ref: React$ElementRef<ElementType>) => React$Node,
```

Comment:
This is not really a render prop since it's not passed at render time nor a prop. It's more like, just `renderFn` or `render` maybe?

Reply:
Good suggestion. I like `renderFn`